core/frontend/src/libs/MAVLink2Rest/index.ts

```diff
@@ -95,6 +97,67 @@ class Mavlink2RestManager {
     return socket
   }
 
+  /**
+   * Helper for COMMAND_LONG messages
+   * @param command 
+   */
+
+  sendCommandLong(
+    command: MavCmd,
+    param1: number | undefined = undefined,
+    param2: number | undefined = undefined,
+    param3: number | undefined = undefined,
+    param4: number | undefined = undefined,
+    param5: number | undefined = undefined,
+    param6: number | undefined = undefined,
+    param7: number | undefined = undefined
+  ) {
+    mavlink2rest.sendMessage({
+      header: {
+        system_id: 255,
+        component_id: 1,
+        sequence: 1,
+      },
+      message: {
+        type: MAVLinkType.COMMAND_LONG,
+        param1: param1 ?? 0,
+        param2: param2 ?? 0,
+        param3: param3 ?? 0,
+        param4: param4 ?? 0,
+        param5: param5 ?? 0,
+        param6: param6 ?? 0,
+        param7: param7 ?? 0,
+        command: {
+          type: command,
+        },
+        target_system: autopilot_data.system_id,
+        target_component: 1,
+        confirmation: 0,
+      },
+    })
+  }
+
+
+  waitForAck(command: MavCmd, timeout: number = 3): Promise<any> {
```

Comment:
It would be nice to have a documentário or something explaining what is the unit of timeout

Reply:
ping @Williangalvani

Reply:
I renamed timeout to timeout_seconds